streams/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from users.models import User
+from posts.models import Posts
+# Create your models here.
+
+class Stream(models.Model):
+    title = models.CharField(max_length = 50)
```

Comment:
Can you put an option to rename the title?

Reply:
Ya, every field is alterable via database operations. No need to change the model for that.